docs/sources/operators-guide/deploying-grafana-mimir/jsonnet/configuring-autoscaling.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Configuring Grafana Mimir autoscaling with Jsonnet"
+menuTitle: "Configuring autoscaling"
+description: "Learn how to configure Grafana Mimir autoscaling when using Jsonnet."
+weight: 30
+---
+
+# Configuring Grafana Mimir autoscaling with Jsonnet
+
+Mimir Jsonnet supports autoscaling for the following components:
+
+- [Querier]({{< relref "../../architecture/components/querier.md" >}})
```

Comment:
should the ruler-querier be listed separately on this list?

Reply:
Ruler-querier autoscaling is not a thing in Mimir OSS jsonnet yet. @ortuman or @stevesg could you add to your TODO list to upstream it (no hurry), once you feel comfortable with that?